cvIgnacioCalahorro.html

```diff
@@ -89,30 +83,28 @@ <h2 class='about'>ABOUT ME:</h2>
                 <p class="ses">Studies</p>
                 <div class="texto-justificado">
                     <p class="unis"> 
-                        Msc GIS at Esri Spain 
+                        MSc in GIS at Esri Spain 
                     </p>
                     <p class="unis1">
-                        &nbsp;&nbsp;&nbsp;&nbsp;Currently. Graduated expected on May 2022
+                        &nbsp;&nbsp;&nbsp;&nbsp;Currently. Expected to graduate in May 2022
                     </p>
                 </div>
                 <div class="texto-justificado">
                     <p class="unis"> 
-                        Msc in Surveying and Mapping.
+                        MSc in Surveying and Mapping at Aalborg University, DK
                     </p>
                     <p class="unis1">
-                        &nbsp;&nbsp;&nbsp;&nbsp;Aalborg University, DK. Graduated 2020 
+                        &nbsp;&nbsp;&nbsp;&nbsp;Graduated 2020 
                     </p>
                     <p>
-                        &nbsp;&nbsp;Activities and societies: Young Professionals in Denmark is a programme that helps international students in Denmark gain insights into the Danish labour market and working culture.
-                        At the first year. Where I was working on the first semester project analyzing a point cloud,
-                        vectorize and compare measurements made with total station with the ones calculate by a code
-                        developed in Python.
+                        &nbsp;&nbsp;Activities and societies: Young Professionals in Denmark is a programme that helps international students in Denmark gain insights into the Danish labour market and working culture. 
+                        I took part in the programme in the first semester of the studies by working on a project which involved point cloud analysis and total station measurement vectorization and comparison. The whole project was carried out in the Python programming language.
```

Comment:
- This sentence overall was messy, I'm not even sure what exactly did you want to say to be honest. I tried to rephrase it, but if I accidentally wrote something else than you wanted, sorry.